### ISSUE TITLE
In the fn list, it's harder to scan return values than args

### ISSUE THREAD

**Issue by maxrothman** [flow-storm/flow-storm-debugger#146]
Right now in the function list view, a fn call's return value only shows if the call's args is selected. The args show in a list in the pane above. If you're looking for a particular call to debug further, it's easy to scan the args list to find the one you want. But if you need to see the return value to see which call you want to debug, you have to go through them one by one.

It might be preferrable to show args and return values in the same list. That way users can scan both sets of values together when looking for a particular call.

Some UI concepts:

Two columns
<img width="536" alt="Screen Shot 2024-02-13 at 11 25 33 AM" src="https://github.com/flow-storm/flow-storm-debugger/assets/2607086/1628c8cf-12bf-463c-a252-c80d545c591a">

Compound row
<img width="322" alt="Screen Shot 2024-02-13 at 11 28 45 AM" src="https://github.com/flow-storm/flow-storm-debugger/assets/2607086/7d767612-87e5-4475-a24a-99e91d8b92a6">

There might be other approaches as well?

The existing lower pane that shows the return values could either be removed entirely, or used to show an expanded/pretty-printed view of the args and return, and provide a way to get to the inspector if desired.

Note: this will somewhat negatively impact performance, since now the return values will have to be serialized ahead of time too.

**Comment by jpmonettas** [comment]
![latest-screenshot](https://github.com/flow-storm/flow-storm-debugger/assets/786299/67ffca49-e4e3-4f85-a161-64329001bcf8)

It now looks like this. The doc was updated also https://flow-storm.github.io/flow-storm-debugger/user_guide.html#_functions_tool

**Comment by maxrothman** [comment]
This is so much better! Could we also get a checkbox for the return value so it can be hidden? I assume the buttons for args and ret open the inspector?

**Comment by jpmonettas** [comment]
Yeah, exactly, those buttons open the inspector where you can then tap and def if needed.
I guess we could add a checkbox yeah, but not sure how much it would help, since it would also need a way  of hiding the args in case you only want to see the returns.

**Comment by jpmonettas** [comment]
@maxrothman I think is better to add another issue for this

**Comment by maxrothman** [comment]
Filed #159
